checkout-ui-custom/src/controller/ViewController.js

```diff
@@ -18,60 +29,48 @@ const ViewController = (() => {
     showMixedProductsMsg: false
   };
 
-  const config = {
-    furnitureId: '0',
-    tvId: '0',
-    simCardId: '0',
-    furnitureForm: {
-      buildingType: ['Free standing', 'House in complex', 'Townhouse', 'Apartment'],
-      parkingDistance: [15, 25, 50, 100],
-      deliveryFloor: ['Ground', '1', '2', '3+'],
-      liftStairs: ['Lift', 'Stairs']
-    },
-    TVorRICAMsg: 'You can\'t collect this order in store because your cart contains items which '
-      + 'require either RICA or TV License validation.',
-    MixedProductsMsg: 'We\'ll ship your furniture and other items in your cart to the selected address. '
-      + 'Only the furniture delivery fee will apply.'
-  };
-
   const checkCartCategories = () => {
+    const { items } = vtexjs.checkout.orderForm;
     const { categories } = CartController.state;
-    const allCategoriesIds = Object.keys(categories);
+    const { config } = CartController;

```

Comment:
I think that can be shortest doing `const {config, state: {categories} } = CartController`